services-geocoding/src/main/java/com/mapbox/api/geocoding/v5/MapboxGeocoding.java

```diff
@@ -581,6 +583,21 @@ public abstract Builder reverseMode(
 
     abstract MapboxGeocoding autoBuild();
 
+
+    /**
+     * Specify the two street names for intersection search.
+     *
+     * @param streetOneName First street name of the intersection
+     * @param streetTwoName Second street name of the intersection
+     * @return this builder for chaining options together
```

Comment:
```suggestion
     * @return this builder for chaining options together
     * @since 4.10.0
```

Reply:
Lots of methods/classes in this particular repo use the `@since` annotation. Easy to forget to add/adjust as you do work in this repo.

It's 4.10.0 here because 4.9.0 is the latest stable version 😃

Reply:
What about the PR #1075 ? I used since 4.9.0 there, should I use 4.10.0 instead?

Reply:
Yeah, use 4.10.0

I see you've already switched it to that number on that PR 👍🏽